packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -192,13 +193,23 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     return this._onRenderContent(tag, buttonProps);
   }
 
+  public componentDidMount() {
+    if (this._isSplitButton && this._splitButtonContainer) {
+      this._events.on(this._splitButtonContainer.value, 'pointerdown', this._onPointerDown, true);
```

Comment:
can't you use react eventing? You should avoid using native events when possible.

Reply:
React does not support this yet, you could potentially mix it in with a `...` but you won't get type safety. Based on this issue https://github.com/facebook/react/issues/499 it looks like there was some strife within the react community as to whether or not pointer events should get added since they still aren't supported by IOS